### ISSUE TITLE
CSS code splitting works now, but not as documented

### ISSUE THREAD

**Issue by hschletz** [nextcloud-libraries/nextcloud-vite-config#222]
While fixing my CSS issue (https://github.com/nextcloud-libraries/nextcloud-vue/issues/5719), I upgraded to 1.3.0 and tried out the new `createEmptyCSSEntryPoints` option, but output was unchanged (i.e. broken).

This feature should only have an effect if the `cssCodeSplit` build option is enabled. According to the Vite documentation, this should be enabled by default unless build.lib is set (which I haven't, at least not explicitly). However, I had to enable it manually in my vite config. Is the documentation wrong, or does this package override the option to FALSE if not set explicitly?

After having done so, `createEmptyCSSEntryPoints` has no effect at all. I can set it to TRUE or FALSE or omit it entirely. Output is correct now in either case.

In summary, all I had to do is:

````ts
export default createAppConfig({
    // multiple entry points
}, {
    config: {
        build: {
            cssCodeSplit: true, // should be TRUE by default?: https://vitejs.dev/config/build-options.html#build-csscodesplit
        },
    },
})
